src/bdd/bdds.jl

```diff
@@ -0,0 +1,1025 @@
+_bdd_nextid = 1
+_bdd_mutex = Threads.SpinLock()
+_bdd_int_typemax = typemax(Int)
+
+"""A Binary Decision Diagram.
+
+- `index`: the vertex variable (-1 if terminal vertex
+- `low`: low child vertex of BDD (undef if terminal vertex)
+- `high`: high child vertex of BDD (undef if terminal vertex)
+- `value`: terminal boolean value
+- `id`: unique identifier
+"""
+mutable struct Diagram
```

Comment:
Why not name it BinaryDecisionDiagram? Is there collision with other things we have.
Unless Diagram can be something else too I guess, so is this limiteed to BDDs only?

Reply:
You're right. Diagram can be kind of confusing. It was originally named that in the BDD.jl package, and frankly I don't know why I chose that name. How about `Bdd` to parallel with Juice's `Sdd`?

Reply:
Sure `Bdd` works as well. Can also have a alias of it as `BinaryDecisionDiagram` so both works.